platform/cri-o/BUILD.bazel

```diff
@@ -8,13 +8,13 @@ load("//bazel:substitute.bzl", "substitute")
 homeworld_deb(
     name = "package",
     bin = {
-        "@com_github_cri_o_cri_o//cmd/crio": "/usr/bin/crio",
-        "@com_github_cri_o_cri_o//conmon": "/usr/libexec/crio/conmon",
+        "@com_github_kubernetes_sigs_cri_o//cmd/crio": "/usr/bin/crio",
+        "@com_github_kubernetes_sigs_cri_o//conmon": "/usr/libexec/crio/conmon",
     },
     data = {
         ":crio.conf": "/etc/crio/crio.conf",
-        "@com_github_cri_o_cri_o//:seccomp.json": "/etc/crio/seccomp.json",
-        "@com_github_cri_o_cri_o//:crio-umount.conf": "/usr/local/share/oci-umount/oci-umount.d/crio-umount.conf",
+        "@com_github_kubernetes_sigs_cri_o//:seccomp.json": "/etc/crio/seccomp.json",
+        "@com_github_kubernetes_sigs_cri_o//:crio-umount.conf": "/usr/local/share/oci-umount/oci-umount.d/crio-umount.conf",
```

Comment:
Because the Bazel<->Go integration has a fixed format it expects for workspace names, which are translations of the URLs. If you don't use them, there are often various problems caused by it where the generated bazel build scripts are incorrect. I believe that's the case here, too.